src/axel_f/excel/coerce.cljc

```diff
@@ -27,3 +28,62 @@
     true "TRUE"
     false "FALSE"
     (str item)))
+
+(defn to-string*
+  "Try to coerce given value to a string type. Returns null for empty value."
```

Comment:
Most other axel-f functions use present tense verbs - 'rounds', 'returns', 'subtracts' etc. The first half of this string would be a better fit as 'Tries to coerce given value to a string type.'